src/battle-animations.ts

```diff
@@ -625,7 +625,7 @@ class BattleScene {
 		pokemonhtml = '<div class="teamicons">' + pokemonhtml + '</div>';
 		const $sidebar = (side.n ? this.$rightbar : this.$leftbar);
 		if (side.name) {
-			$sidebar.html('<div class="trainer"><strong>' + BattleLog.escapeHTML(side.name) + '</strong><div class="trainersprite" style="background-image:url(' + Dex.resolveAvatar(side.avatar) + ')"></div>' + pokemonhtml + '</div>');
+			$sidebar.html('<div class="trainer" ' + (side.rating ? 'title="Rating: ' + side.rating + "\">" : '>') + '<strong>' + BattleLog.escapeHTML(side.name) + '</strong><div class="trainersprite" style="background-image:url(' + Dex.resolveAvatar(side.avatar) + ')"></div>' + pokemonhtml + '</div>');
```

Comment:
This isn't sanitized.

```suggestion
			$sidebar.html(`<div class="trainer" ${side.rating ? 'title="Rating: ' + BattleLog.escapeHTML(side.rating) + '"' : ''}><strong>${BattleLog.escapeHTML(side.name)}</strong><div class="trainersprite" style="background-image:url(${Dex.resolveAvatar(side.avatar)})"></div>${pokemonhtml}</div>`);
```